action.yml

```
@@ -26,6 +26,8 @@ runs:
     - name: Set NABU_IMAGE and Run Dev Mode
       shell: bash
       run: |
+        python3 main.py pull --profiles localInfra production
+        python3 main.py build --profiles localInfra production
         python3 main.py dev --detach
```

```suggestion
        python3 main.py dev --detach --build
```

fyi you can also just add --build to main. might work as well but im not picky great if this works